### DISCUSSION TITLE
Is FreightFrenzy_BCDM.tflite available.

### DISCUSSION THREAD

**Discussion by cherukurir** [FIRST-Tech-Challenge/FtcRobotController#190] [Q&A] [answered]
I am looking for tflite file for the freight frenzy. When are they going to be available?

**Comment by cmacfarl** [comment, marked as answer]
They are served off of maven now and built into the APK when you deploy the robot controller.

If you still want the file for some reason, you may pull the jar that contains it directly off of maven central, unzip the jar, and extract the file, but that's not specifically required to use the model with the existing build setup.

The assets jar is located here: https://repo1.maven.org/maven2/org/firstinspires/ftc/gameAssets-FreightFrenzy/1.0.0/

**Reply by cherukurir** [reply]
Thank you that helps a lot.
